pkg/config/api-server/catalog/config.go

```diff
@@ -2,14 +2,15 @@ package catalog
 
 // Not yet exposed via YAML and env vars on purpose
 type CatalogConfig struct {
-	Bootstrap            BootstrapApiConfig
-	DataplaneToken       DataplaneTokenApiConfig // DEPRECATED: remove in next major version of Kuma
-	Admin                AdminApiConfig
-	MonitoringAssignment MonitoringAssignmentApiConfig
+	Bootstrap            BootstrapApiConfig            `yaml:"bootstrap"`
+	DataplaneToken       DataplaneTokenApiConfig       `yaml:"-"` // DEPRECATED: remove in next major version of Kuma
+	Admin                AdminApiConfig                `yaml:"-"`
+	MonitoringAssignment MonitoringAssignmentApiConfig `yaml:"monitoringAssignment"`
+	Sds                  SdsApiConfig                  `yaml:"sds"`
 }
 
 type BootstrapApiConfig struct {
-	Url string
+	Url string `yaml:"url" envconfig:"kuma_bootstrap_server_public_url"`
```

Comment:
We name the env vars the same as YAML path, so the env should be `KUMA_API_SERVER_CATALOG_BOOTSTRAP_URL`
other envs:
`KUMA_API_SERVER_CATALOG_MONITORING_ASSIGNMENT_URL`
`KUMA_API_SERVER_CATALOG_SDS_URL`